### PR TITLE
User organization association

### PR DESCRIPTION
**Pull request by mcsearchin**

### CODE REVIEW

File: lib/discovery_api_web/controllers/visualization_controller.ex [hunk 1]
```diff
@@ -26,7 +26,7 @@ defmodule DiscoveryApiWeb.VisualizationController do
   end
 
   def update(conn, %{"id" => public_id} = attribute_changes) do
-    with {:ok, user} <- Users.get_user(conn.assigns.current_user),
+    with {:ok, user} <- Users.get_user(conn.assigns.current_user, :subject_id),
```
Comment (bbrewer-papa-x) [inline lib/discovery_api_web/controllers/visualization_controller.ex]: I like this. Was thinking of doing this for visualizations back in the day.

File: test/unit/discovery_api/data/mapper_test.exs [hunk 2]
```diff
@@ -7,7 +7,13 @@ defmodule DiscoveryApi.Data.MapperTest do
 
   describe "to_data_model/2 hard overrides" do
     data_test "with #{inspect(overrides)} should default #{inspect(field)} to #{inspect(value)}" do
-      dataset = TDG.create_dataset(overrides)
+      dataset =
+        overrides
+        |> TDG.create_dataset()
+        # NOTE: *for now* we need to re-apply the overrides to make sure that our provided source format does not get converted to a mime type
```
Comment (bbrewer-papa-x) [inline test/unit/discovery_api/data/mapper_test.exs]: Snap

File: lib/discovery_api/event_handler.ex [hunk 3]
```diff
@@ -1,12 +1,25 @@
 defmodule DiscoveryApi.EventHandler do
   @moduledoc "Event Handler for event stream"
+
   use Brook.Event.Handler
-  import SmartCity.Event, only: [organization_update: 0]
+  import SmartCity.Event, only: [organization_update: 0, user_organization_associate: 0]
+  require Logger
   alias SmartCity.Organization
+  alias SmartCity.UserOrganizationAssociate
```
Comment (jdenen) [inline lib/discovery_api/event_handler.ex]:
```elixir
alias SmartCity.{Organization, UserOrganizationAssociate}
```

File: lib/discovery_api/event_handler.ex [hunk 4]
```diff
@@ -1,12 +1,25 @@
...
   alias DiscoveryApi.Schemas.Organizations
+  alias DiscoveryApi.Schemas.Users
```
Comment (jdenen) [inline lib/discovery_api/event_handler.ex]:
```elixir
alias DiscoveryApi.Schemas.{Organizations, Users}
```

**Pull request merged**